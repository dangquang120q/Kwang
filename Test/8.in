36
43
6383535 1107111 3540023 3494048 9822150 892132 8269640 3110208 2085194 5516712 9407510 7594994 4605407 9049404 9557997 2100393 2792512 1168851 9500286 5450621 9416137 379357 8120821 1941323 1780990 4011468 474386 5326382 5505719 5736468 9348071 6813931 3375256 5858275 7291134 2502105 6932129 1975339 4818415 7140494 1751971 1130805 3029778 
87
5834603 7219257 9853651 6385949 806929 6202975 253765 6231583 954529 3804611 5162588 4182726 9864163 8697923 5001741 3087350 661042 3636785 1158579 7197542 8064209 8409219 7529455 451927 1223330 164170 459885 2508285 2169397 2641952 3569334 6899781 5349805 4032239 3347712 3248583 4513875 2171213 6235654 953480 4171566 4975259 1510505 8135246 1641611 1384448 1967336 7939526 4830690 1029523 1715242 1171751 241160 4599504 8535244 2162312 7023801 2886291 394145 6865526 3205920 1448038 1338422 4399774 3361940 1167586 3048721 1434344 3507291 8126443 8680853 681787 8250537 4846245 6294291 1916714 4686613 3314293 7300305 3845121 5463893 948108 1224218 2802465 9944731 1370291 5831174 
12
1553201 6177365 2954143 7150578 9230348 5727672 419758 3923109 9120541 2051466 7163243 9874568 
57
6990522 8549063 7477638 389646 1208111 7040919 1961870 309819 1413584 6768243 4106486 3071356 5942277 7696810 1598773 8032172 7370172 4152617 1463859 1653343 2172057 9882256 7530186 542042 3098475 9752370 3612750 9707581 9615204 5403031 4313327 9907270 2487816 8137709 124540 6310788 139398 7302611 2648006 1027062 6090587 8236485 5168559 9967452 2758257 6826291 6739903 1844010 7618027 1706208 3633669 506735 4782311 2788410 8866326 3318993 1988051 
16
6891362 4676100 182093 4422816 161706 6028161 7838271 1975541 6960727 3887462 6122967 7942673 1701401 1659452 9799728 1971310 
28
2417288 3786690 6512364 7732729 3395990 3620706 7534694 2795749 4832217 4592143 4964497 3153024 9338415 2921522 9971501 9134096 864743 5984630 2920633 7678115 4705783 5978203 9503536 4220046 1393793 2245582 9694701 8962280 
65
9996091 3285201 8862817 1664091 1547401 8595151 9646939 6434681 2153229 749516 2786512 5307080 5905771 6575992 7107099 6363460 557913 8353429 7727275 5334212 5583316 2882985 3325964 6868147 1760650 8046732 3686923 5740335 2912735 5773877 8046609 7875959 6257031 2163926 3379214 1124985 635789 8573743 7852713 8099535 9819599 8574893 4514879 7107630 2639819 6628456 9679417 6561869 6273519 7807975 7029489 4093300 1270043 3442310 6415611 7695015 8017455 4047786 1681129 9002818 2839737 9287799 3151789 4329262 5213948 
13
2185886 7860223 6534835 6594771 1263142 4168435 5801179 3107183 1605392 8480359 5452604 2715522 4744899 
96
7910878 4344679 9258827 1468420 6220862 7203874 9148691 5088038 8207205 3880533 9159155 8958710 5228402 4892879 1757735 7123943 2446034 4249943 7236604 2788085 8657823 5605595 208774 6158837 723093 3775146 1275668 132723 9728985 5887181 552937 145964 2520928 4315962 6218040 8808262 6915774 368770 9778182 2467650 2280714 5138753 8301897 9928544 2321160 7532179 255284 9174438 1365809 7502367 3939234 6178915 1170508 7806212 1688284 6315448 3225733 5136436 7241087 8474136 2314038 9917388 2830003 2396624 9285125 5640480 3250723 3270835 4463349 5188852 7467963 7359629 4464845 5571155 9124001 6283005 3758533 3977107 1697248 6473159 7931646 3982449 1924536 4122335 1237138 135174 9027773 4959601 1737115 8207162 250447 5128860 313196 7368942 8325571 9818831 
30
5778092 9749670 7093504 5076592 3167197 3406980 6778249 671576 6230943 5385128 4475985 5173599 4554588 4525451 9951495 6969136 1446100 4324776 7745787 1337658 6069289 4609564 9305000 841859 6761972 4455420 1518503 2039453 7164741 5373281 
66
5405180 1330562 7892770 5015181 8205030 5565811 9486611 8385463 3720323 3833796 1999769 8309220 2919931 5539837 5393270 481994 8562782 9210024 8323531 7343481 5629370 7120694 8106366 6942857 5553792 6432215 1421658 9978026 1978383 8776913 4717734 2467241 2870492 3095168 7556405 2491639 1558960 557941 4186489 2059720 3699145 3790172 8886643 8598873 1807122 3563170 5904691 8694912 3627784 2540792 1356890 6883791 2130415 2179103 1747568 7017196 637346 4281258 3737678 876568 9779243 1197150 1263056 1088677 9502065 4302971 
33
6801278 2226807 6632247 5694630 7721044 403882 4371838 2118972 3940198 8840060 9973042 4438102 4844083 8816745 2892220 2754441 7005584 1626024 3770588 6411160 606441 4484056 7383226 9898458 4018539 9890320 8134385 3181602 3043969 8033693 6335944 4787442 1256993 
8
1106649 7142052 169029 4783464 6274978 5896153 4255340 1414512 
44
9468542 6396050 8754131 9634909 7838729 2434506 7256960 1626126 3901250 6338347 4091074 715407 4582700 1245482 2553780 4588964 9002202 9586920 924700 578929 9678603 1096968 7559552 9819514 851287 2806965 4829123 4480728 7195492 2544742 4828956 6621177 9137468 4261769 4132795 191828 1438780 1906091 8848933 9689600 6247669 8777114 5684317 5218579 
43
8072926 4877049 7897520 803979 4429106 3853474 6095075 4494426 2118530 6855484 6309107 5207599 4811222 2952536 6790198 4982567 2952255 2551318 4555087 9846360 9540063 7451036 2626788 866294 1772788 9987846 193387 1635538 7169802 3971454 7215462 7295447 2247431 258930 3347279 5084716 881180 9016349 3224012 3778782 8681020 673374 9622264 
87
9843098 9414625 3315583 9049616 399346 9746554 4678739 6265956 8641659 4332495 9053767 1814912 372759 982929 6069441 3201142 2499065 8007799 9413513 6993884 4070098 6782832 3162019 817014 3468154 4191259 914680 9792622 3069205 6073557 5515384 2396721 5419365 1476153 9803070 2860620 3798582 3540329 9754055 8566059 8602745 5541788 3425183 9009233 8427323 2575662 709840 3014904 188649 7431517 2007390 5645091 3803016 6448037 6387583 9469688 2618310 4853606 1481862 4537620 2423463 2815881 1504528 4204237 1783913 6779736 3517695 4888841 9619458 8232867 9650039 6021197 977450 8792536 3403129 8538876 9408719 2316695 8362825 8964891 1594673 2682485 2762535 4243973 3725385 1489275 1455969 
22
3240973 2899961 5541343 9671059 868349 868304 3038927 916453 1470482 944813 3010027 9012683 3312918 6352235 4521560 4514529 8904921 7731800 5032543 3296647 6748764 4983744 
23
7304677 1267013 1195803 1021287 8868823 8366833 2291181 9615250 7194374 3826538 8140991 9493582 6572363 9169093 6200904 5092488 4046912 1525789 9254963 3063890 4550151 4010795 4258165 
13
6753057 2847339 8605836 7143297 4012204 9130608 6038637 7639130 4717091 998920 9115968 9104411 650254 
19
8631476 7978337 991398 9105109 5050063 9554787 7231188 8408313 4079059 5200366 1490185 5139673 6381227 4777183 241940 4672883 6722105 5037518 5485077 
40
615931 1015685 9311684 2354766 1708688 6169417 5348790 7896059 2602272 9661226 3680082 5859671 2029780 6248327 6248272 1856170 7683959 1996045 7534072 8333199 3238152 8243413 5412086 4543005 8596994 4201085 2959150 3101399 1796745 8937260 7653025 2736319 3914414 732655 7822378 2852223 1534354 1720594 4692427 4478456 
35
9349058 1523942 6564722 4059344 4204428 8135124 574214 9807299 8471062 3028789 5319935 8221092 7306453 6474077 9685307 7253295 4898079 7563179 4599689 5285690 3880172 8272786 4182052 5423920 3924773 5001702 4152666 9283244 3428697 2609019 3957282 2443584 1115646 8619511 8574982 
37
6056261 190061 5533686 4786382 5678995 6488882 7959707 6791205 3413705 4469533 8940652 5354318 9817632 2110094 3437181 5762208 9020632 2393508 1742737 4154185 1250766 2885668 4387176 7961431 8450734 7425732 8717773 9985912 5983993 1391787 6892224 6213891 3581621 8312099 5917550 9118196 8764287 
28
3264120 6757065 6367579 5783821 5696851 2330094 9873859 1858023 8144590 5622865 372930 9674520 4915628 6539906 3761692 7406415 6391400 5391471 4917282 8018247 9612143 6765346 539139 637345 4559314 8006477 9006348 8657068 
89
1880261 2936990 1544533 9446955 4069471 2454039 9825214 286227 5456773 4006781 7872109 4143938 1703407 6966585 2081209 8262653 2176571 6890917 2314685 4426267 1186010 2907969 5336821 3177002 196407 5199010 5117182 7384761 1777694 6178220 3388877 6634048 8107630 3425112 9651831 966369 435548 7992888 4048381 2241536 7631073 6879136 5451849 5045336 8920696 5027176 8336768 6864034 5517157 9400364 2324506 8293448 1032109 1652339 3792551 5576121 2617299 1324402 5012027 4439298 1895773 9025299 4833960 4669792 904254 2108400 8968322 502277 2123006 1891735 9005548 9236837 9397885 3148967 6094987 5236060 5092916 7093008 2432054 4608065 9031256 6481743 7825934 6223765 5493686 6258051 9234878 6859751 1856305 
80
2986655 7848636 9419987 7799738 528882 959687 1656328 3991185 6785733 6631922 4971730 1794895 9243400 8549463 6362908 8084684 5703134 6423927 2947813 4729859 1533470 1625453 7600094 3967698 2091896 1119270 3626372 253194 9395096 9720754 7683931 8769107 9271537 7537837 8591736 2277598 4128353 4917971 1382418 5081679 5351992 180185 7766552 7548637 3640575 176136 1479398 7419216 2045523 3210720 982522 1647179 5292471 8725075 8905455 1146518 1180816 3493104 3188725 3695709 3877851 8307520 5616335 3573311 3764188 2991814 602926 6574442 1753584 8438588 4382892 2672238 884257 1557326 4222856 2930835 7010239 9886038 4568661 8478239 
3
3148284 1172950 1918499 
64
5204502 7058465 4896748 8444814 6412898 3021132 1163139 8756108 4713393 7982029 3131223 3683440 3975268 7173773 9173458 5089227 2821067 6370247 920261 2854882 9155583 9628020 8446260 9425221 2122258 5652367 4200374 382671 4416072 1912561 9774838 9117861 7445880 7580261 1757348 7005349 4736244 9794811 4272947 6014199 9598636 1727122 2851423 6264069 8264422 9790706 8968577 490068 1643695 2761386 7002817 9162302 709396 8998486 7610082 8089540 9055025 6036933 2199914 8348051 735378 8612851 9908014 9195121 
64
5597345 9316752 7488490 1305315 5237081 4720367 2251801 380500 1711686 4896226 5399177 5774870 5964345 8505330 4786378 475374 6173769 3575070 4998042 5062756 732407 4142027 6048643 906330 7128867 1799469 2075449 6515743 6737581 5520586 3729778 175174 8833825 2906583 8950957 3616369 8777578 978169 8643307 4897163 5303558 8214527 6535288 9599547 5587508 2253742 509931 8040751 3596805 4335105 553443 5628663 7379748 8003602 7999413 4763360 2990527 5488297 7157801 3541182 7769182 7061424 723145 5969990 
37
2940441 9842788 3299522 7148900 4791440 4972885 7108892 4248668 5649145 9424580 9208230 2524585 2130665 7527701 8232857 5360529 8667962 2766547 1643916 2187109 7075599 6912894 6364595 4719242 9539277 7606812 392395 1137287 1050071 2975019 7457014 5015666 1591134 3581702 2371184 6482915 9678675 
14
9002839 2753761 693280 7214657 3672819 695563 9152778 2274612 3351912 7335110 1328381 7522209 2874684 452286 
61
6510111 345298 3058013 7410873 4158593 6888532 7842292 427265 9263458 8718220 326006 6790383 3483940 6705638 4764811 7711422 7809257 7369989 7440576 7962928 3646184 5013381 3231651 6530845 1972123 6472011 237749 2446152 8090932 2197582 7752656 8846340 3932831 7874979 1591331 3642992 7169828 7630142 8658197 415712 9793179 7495098 7075654 9879510 2319645 9202128 7381158 4775119 9850174 9791006 1338252 7147260 7850357 5739896 5871329 4084506 4840951 153771 9491492 8563904 4567788 
8
8933576 8485663 8178864 1770444 3281972 4751567 5270686 3960538 
91
828238 6128539 2108116 2166324 6323761 6360896 8932489 1953931 1517282 6967466 5989303 9784797 4385256 9352607 8367090 578913 7748611 510884 8236646 866487 8907002 9995488 2264797 4443018 5090176 4731400 5445664 4587120 6889616 9118771 1896158 7558442 6392267 4960222 7733169 5106740 3303650 4449509 8944364 191140 8623570 9146469 1540041 572924 5058214 8226739 4991487 9533696 5272807 8511966 5642669 5461464 3114609 8710765 557205 8340668 3411277 8532884 854755 6424271 7629631 4585958 7933454 5648146 9474424 3426628 1239775 3401144 6596947 2427546 6091193 7980330 7798787 6788793 7176824 1439602 5304585 2445495 6066381 2647401 1520220 719843 5156519 1216033 7776794 4722026 8804600 4916733 5505063 6402573 3739254 
66
4976813 3481401 6973718 8116482 3545346 1221576 9243647 6257760 3192328 3742404 7752634 2966638 2557375 8636455 4018494 1458877 9518069 4680306 6565492 5898526 2313986 7253060 6145869 2120941 6406551 5702390 490729 903408 6471974 2698277 4877720 5551804 269351 8039547 8783926 4581606 6706562 9762073 9307603 6672773 9632675 9897956 5594798 9239641 5245778 7328567 2799638 1192611 1587596 4476800 630973 3034713 4108462 1714521 5631754 6065506 4073840 1231692 9081916 1961421 6418456 1677036 2693248 8557169 1891723 2958626 
4
1594414 1873849 5645105 8037064 
