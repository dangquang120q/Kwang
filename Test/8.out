28880433
29662545
28225457
29756978
25580672
29169738
29495107
22935353
29664763
29006165
29259334
29761820
19313183
29187717
29456470
29438790
27588663
28363795
27350987
27291372
27910170
28841664
28921740
29160522
28924000
29026779
6239733
29493531
27867256
29060740
25677826
29522863
25598103
29313981
29292704
15556018
